src/community/release/ext-mbstyle.xml

```diff
@@ -0,0 +1,18 @@
+<assembly>
+  <id>mbstyle-plugin</id>
+  <formats>
+    <format>zip</format>
+  </formats>
+  <includeBaseDirectory>false</includeBaseDirectory>
+  <fileSets>
+    <fileSet>
+      <directory>release/target/dependency</directory>
+      <outputDirectory></outputDirectory>
+      <includes>
+        <include>gs-mbstyle-*.jar</include>
+        <include>gt-mbstyle-*.jar</include>
+        <include>hamcrest-all-*.jar</include>
```

Comment:
I am not used to seeing hamcrest included, that is usually for test cases

Reply:
Looks like this was incorrectly scoped as compile in gt-mbstyle: https://github.com/geotools/geotools/blob/master/modules/unsupported/mbstyle/pom.xml#L130-L134

I'll update here and in geotools...